README.md

```diff
@@ -1,6 +1,6 @@
 # Project Scanner
 
-Scans repository for iOS, macOS, Android, Xamarin, Fastlane and Cordova projects.
```

Comment:
NOTE: ionic and react native is missing from the list

Reply:
NOTE: README could be auto-generated using the [generate_readme wf](https://github.com/bitrise-steplib/steps-xcode-archive/blob/master/bitrise.yml#L45).

Reply:
I have added the readme generation workflow and also generated a new readme.